packages/shared/lib/models/Sync.ts

```diff
@@ -259,11 +259,12 @@ export interface SyncConfigWithProvider {
 }
 
 export interface RunScriptOptions {
+    syncConfig?: SyncConfig;
     syncName: string;
     syncId: string;
     activityLogId: number | undefined;
     nangoProps: NangoProps;
-    integrationData: NangoIntegrationData;
+    integrationData: Pick<NangoIntegrationData, 'fileLocation'>;
```

Comment:
This will be removed in next PR, but since we do multiple steps deployment it's safer not to modify runner signature